internal/controller/standalone_pgadmin/postgrescluster.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 Crunchy Data Solutions, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package standalone_pgadmin
+
+import (
+	"context"
+
+	"github.com/crunchydata/postgres-operator/internal/naming"
+	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//+kubebuilder:rbac:groups="postgres-operator.crunchydata.com",resources="pgpgadmins",verbs={list}
+
+// findPGAdminsForPostgresCluster returns PGAdmins that target a given cluster.
+func (r *PGAdminReconciler) findPGAdminsForPostgresCluster(
```

Comment:
where is this used?

Reply:
That's used in the `watchPostgresClusters` func, which is used by the `SetupWithManager`.

I basically followed the form of our upgrade controller here: if a pgadmin changes, we queue that pgadmin to reconcile; but if a postgrescluster changes (or is added/deleted), then we check if that postgrescluster would be selected by any existing pgadmins, and if found, we queue any pgadmins to reconcile.